install/fossinit.php

```diff
@@ -166,6 +166,11 @@ function explainUsage()
   require_once("$LIBEXECDIR/resequence_author_table.php"); // If table exists, clean up for Schema
 }
 
+if($isUpdating && substr(str_replace(".", "", $sysconfig['Release']), 0, 3) <= 330)
+{
+  require_once("$LIBEXECDIR/dbmigrate_3.3-3.4.php");
+  Migrate_33_34($dbManager, $Verbose);
```

Comment:
Shouldn't the `Migrate_33_34` call be later in the script, e.g. after the older migration scripts? otherwise one would have to call the script multiple times?

Reply:
But this means, that if you migrate from 3.0.0 -> 3.4.0 you have to call the script multiple times?

Reply:
fixed by: https://github.com/fossology/fossology/pull/1210#discussion_r220063278